firmware/make_t85_default.cmd

```diff
@@ -0,0 +1,20 @@
+@echo off
```

Comment:
why is this script needed? The default config can be made by "make"

Reply:
I used this script heavily for testing, since building all takes way longer until the default (reference for me) is built.

Reply:
just use "make" ? It will default to t85_default.

Reply:
You are right!
I personally like to have a script and not to open a CMD window, navigate to the right directory and then type make.
This script is the one click solution like the old make_all.sh (it has vanished... OMG)

Reply:
You could also use your IDE to invoke build?

Reply:
anyways, I think it would be better not to include environment specific scripts into the main repository. the *.cmd will fail under anything but windows.

Reply:
There are more than one makefile :-(

Reply:
There is only one?!?

Reply:
one in firmware, one in commandline?

Reply:
but they are in different folders ?!?

Reply:
What is about the makerelease.bat script in the [2.04 release](https://github.com/micronucleus/micronucleus/releases/tag/2.04)?

There are other projects which supply these make helper scripts
Here examples from avrdude
https://github.com/mariusgreuel/avrdude/blob/windows/tools/build-mingw32.sh
and 
https://github.com/mariusgreuel/avrdude/tree/windows/windows

and Arduino
https://github.com/arduino/Arduino/blob/master/build/build_all_dist.bash

Reply:
Remove commited